notebooks/2023-03-21-single-country-rollouts/mm/1_mm_train_model.ipynb

```diff
@@ -0,0 +1,883 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    country_data[['label']] = scaler().fit_transform(country_data[[label_col]])

</pre><p>Got it on keeping scaling for labels but I think it will still give same predictions distribution shape if raw labels are used. Agree to keep to make prediction range of values consistent with cross country but may need to update these with same scaler from the new cross country models we're testing out if ever they produce better results (e.g. MinMaxScaler) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/thinkingmachines/unicef-ai4d-poverty-mapping/pull/173/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
Aligned on keeping the same scaler with cross-country model for uniformity.